util/src/main/java/org/datacommons/util/SummaryReportGenerator.java

```diff
@@ -47,6 +48,8 @@ public static final class StatVarSummary {
     Set<String> observationPeriods = new HashSet<>();
     // This set will not be populated when StatVarSummary is generated by PlaceSeriesSummary
     Set<String> dates = new HashSet<>();
+    double minValue = Double.MAX_VALUE;
+    double maxValue = Double.MIN_VALUE;
```

Comment:
If all values for an SV are measurementResult (not a number), would the summary have these initial values?
Should we reset to empty in the final report if they have the initial values?

Reply:
I am updated the code to return a NaN value for such cases.